docs/4.1. Node to Authorization Server Interactions.md

```diff
@@ -189,12 +189,12 @@ Content-Type: application/json
 ```
 
 ## Refreshing Bearer Tokens
-The Bearer Token has a limited life specified by the value of the `expires_in` parameter in seconds. The Node should refresh the token before [it exceeds its half life](https://specs.amwa.tv/is-10/branches/v1.0-dev/docs/4.4._Behaviour_-_Access_Tokens.html#access-token-lifetime) (if the token lifetime is 30 seconds, then it should be refreshed at least 15 seconds before token expiry).
+The Bearer Token has a limited life specified by the value of the `expires_in` parameter in seconds. The Node needs to refresh the token before [it exceeds its half life](https://specs.amwa.tv/is-10/branches/v1.0-dev/docs/4.4._Behaviour_-_Access_Tokens.html#access-token-lifetime) (if the token lifetime is 30 seconds, then refresh it at least 15 seconds before token expiry).
 
 Unlike the Authorization Code Flow, the Client Credential Flow does not issue a refresh token; 'refreshing' the token simply involves repeating the Client Credential Flow.
 
 ## Fetching Authorization Server Public Keys
-As well as acquiring a token to allow this Node to register, the Node will also need the [Authorization Server's public keys](https://specs.amwa.tv/is-10/branches/v1.0-dev/docs/4.5._Behaviour_-_Resource_Servers.html#public-keys) so that it can authenticate calls on its own APIs. These keys should be fetched every hour to ensure the keys are always up to date.
+As well as acquiring a token to allow this Node to register, the Node will also need the [Authorization Server's public keys](https://specs.amwa.tv/is-10/branches/v1.0-dev/docs/4.5._Behaviour_-_Resource_Servers.html#public-keys) so that it can authenticate calls on its own APIs. These keys are fetched every hour to ensure the keys are always up to date.
```

Comment:
Just a suggestion... 
```suggestion
As well as acquiring a token to allow this Node to register, the Node also needs the [Authorization Server's public keys](https://specs.amwa.tv/is-10/branches/v1.0-dev/docs/4.5._Behaviour_-_Resource_Servers.html#public-keys) so that it can authenticate calls on its own APIs. These keys need to be fetched every hour to ensure the keys are always up to date.
```

Reply:
LGTM